nasa-gibs/worldview
# WV-1934: Migrate config properties to GIBS metadata

## Description

* Removed the following properties from layer configs and bring in their GIBS metadata equivalent
  * title
  * subtitle
  * daynight
  * inactive (ongoing)
  * tracks (orbitTracks)
* Remove code deriving a `facetPeriod` property and use GIBS metadata `layerPeriod` property instead
* Update layer config JSON schema
* Update layer config docs

## To test

1. `npm run build`
2. Confirm orbit track layers work as expected
3. Confirm layer filtering works as expected, especially by daynight and layer period.
